app/views/purchases/_form.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= form_with(model: purchase, url: {action: "create"}) do |form| %>
+    <% if purchase.errors.any? %>
+      <div style="color: red">
+        <h2><%= pluralize(purchase.errors.count, "error") %> prohibited this purchase from being saved:</h2>
+        <ul>
+          <% purchase.errors.each do |error| %>
+            <li><%= error.full_message %></li>
+          <% end %>
+        </ul>
+      </div>
+    <% end %>
+  <div >
+      <%= form.text_field :name, type: "text", placeholder: "name", class: "form-control" %><br />
+      </div>
+      <div >
+
+      <%= form.text_field :amount, type: "number", placeholder: "Amount", class: "form-control" %><br />
+             </div>
+      <button class="btn btn-primary">
+      <%= form.submit "Add Purchase" , class: "add-new"%>
+      </button>
```

Comment:
[OPTIONAL] Because the previous reviewer didn't notice this the following suggestions are NOT mandatory. I do recommend strongly for you to have a look at it.
- [ ] The form is missing a field where users can select the categories to which the transaction should belong. I recommend adding a field to the form that allows the user to choose at least one category (the user should be able to choose more than one category).
- [ ] In the splash screen, the app should have a simple page with the name of your app with links to the sign up and log in pages.
- [ ] As a full-stack engineer, you are required to follow the design guidelines. The Product Designer is the only one with the authority to alter the design.